reboot.ks.in

```diff
@@ -0,0 +1,23 @@
+#version=DEVEL
+#test name: reboot
+
+# Use defaults.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+# Reboot the installed system.
+reboot
+
+# Validate on the first boot.
+%ksappend validation/success_on_first_boot.ks
+
+%post
+
+# Create a script with the first boot test.
+# Print errors to stdout.
+cat > /usr/libexec/kickstart-test.sh << EOF
+journalctl --priority=err --quiet
```

Comment:
The test will fail due to any error in journal (for example from kernel) - is it really wanted?

Reply:
Fixed.